.github/pull_request_template.md

```diff
@@ -1,12 +1,15 @@
-<!-- Please provide a descriptive title for your Pull Request above.  -->
```

Comment:
My hunch is that, however it's written, this will never be read given it's location. Yay, nay?